### DISCUSSION TITLE
same topic name in different namespace

### DISCUSSION THREAD

**Discussion by wangjia007bond** [strimzi/strimzi-kafka-operator#6730] [General]
can we create same topic name in different namespace, is it a good approach?
From client how to distinguish this two topic and how to connect to these topics?

**Comment by scholzj** [comment]
The Topic Operator always watches only one namespace. So no, you cannot do that using the `KafkaTopic` resources.

**Reply by wangjia007bond** [reply]
Thanks, I have install Kafka in Kafka namespace and topic operator, what do you mean topic operator only watch one namespaces. Can I create topic in other namespace like dev namespace?

**Reply by scholzj** [reply]
The Topci Operator watches always only one namespace. By default, it is the namespace where the KAfka cluster was deployed, but you can change it. But it will still be only one namespace. In that namespace, you can create your KafkaTopic resources. If you create them in any other namespace, they will be ignored.
